examples/places-service-with-element/components/places-service-with-element/places-service-with-element.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {useEffect, useRef, useState} from 'react';
+
+import {useGoogleMap, usePlacesService} from '@ubilabs/google-maps-react-hooks';
+
+import styles from './places-service-with-element.module.css';
+
+const PlacesServiceElement = () => {
+  const map = useGoogleMap();
+
+  const divRef = useRef<HTMLDivElement>(null);
+
+  const [placeResults, setPlaceResults] = useState<
+    google.maps.places.PlaceResult[]
+  >([]);
+
+  // Get the places service from the usePlacesService hook
+  const service = usePlacesService({divElement: divRef.current});
+
+  useEffect(() => {
+    if (!divRef || !service) {
```

Comment:
Is it important that the div is already available here? If so it should be added to the dependency array as well.
```suggestion
    if (!service) {
```